tests/lib/perf.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set +x
+
+execution_time_ms(){
+    command="$1"
+    ts=$(date +%s%N)
+    ( $command ) > /dev/null 2>&1
```

Comment:
Producing (or not) output can influence the time it takes to execute. I think you can drop `>/dev/null 2>&1` and let the caller do this in the command string they pass.

I think you want something like:
```
( eval "$command")
```
Otherwise this will fail: `execution_time_ms "ls -l > /dev/null 2>&1"`

Reply:
Could we use something like 
```
command="$1"
summary="$2"
echo "perf for $summary" >> $RES
/usr/bin/time -f %E --output $RES --apend $command
```
this way we avoid mixing stdout with the performance results?